src/Model/Resolver/ProductResolver.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/quote-graphql
+ * @link https://github.com/scandipwa/quote-graphql
+ */
+
+declare(strict_types=1);
+
+namespace ScandiPWA\QuoteGraphQl\Model\Resolver;
+
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Catalog\Model\ProductRepository;
+
+/**
+ * Retrieves the Product list in orders
+ */
+class ProductResolver implements ResolverInterface
+{
+    /**
+     * @var ProductRepository
+     */
+    protected $productRepository;
+
+    /**
+     * @param ProductRepository $productRepository
+     */
+    public function __construct(
+        ProductRepository $productRepository
+    ) {
+        $this->productRepository = $productRepository;
+    }
+
+    /**
+     * Get All Product Items of Order.
+     * @inheritdoc
+     */
+    public function resolve(Field $field, $context, ResolveInfo $info, array $value = null, array $args = null)
+    {
+        if (!isset($value['products'])) {
```

Comment:
We can avoid this check if we set this argument as non-nullable in graphql schema.

Reply:
Can you, please, show me, from where I can get access to this argument.
Is it ```order_products: [ProductInterface] @resolver(class: "\\ScandiPWA\\QuoteGraphQl\\Model\\Resolver\\ProductResolver")```
where I should replace `[ProductInterface]` with `[ProductInterface!]` ?

Reply:
My mistake. Let's leave as is.